src/apps/concepts/pages/ViewConceptsPage.tsx

```diff
@@ -142,6 +142,7 @@ const ViewConceptsPage: React.FC<Props> = ({
     q: initialQ = "",
     classFilters: initialClassFilters = [],
     dataTypeFilters: initialDataTypeFilters = [],
+    generalFilters: initialGeneralFilters = [],
```

Comment:
@jwamalwa  well done julie,are you able to trace this (Property 'generalFilters' does not exist on type 'OptionalQueryParams'.) as per your ci logs?

Reply:
@HerbertYiga thanks for the review but my real work is on this link https://github.com/openmrs/openmrs-ocl-client/pull/673. This brunch needs to be closed after reverting the changes. Once again thanks.